src/test/java/no/sikt/sws/testutils/TestConstants.java

```diff
@@ -2,9 +2,13 @@
 
 public final class TestConstants {
 
-    public static final String TEST_SCOPE  = "https://api.sws.aws.sikt.no/scopes/workspace-sondre";
     public static final String TEST_INDEX_1  = "some-index-1";
-    public static final String TEST_WORKSPACE_PREFIX = "workspace-sondre-";
+
+    public static final String TEST_SCOPE_SONDRE = "https://api.sws.aws.sikt.no/scopes/workspace-sondre";
+    public static final String TEST_PREFIX_SONDRE = "workspace-sondre-";
+
+    public static final String TEST_SCOPE_MOCKNAME = "https://api.sws.aws.sikt.no/scopes/workspace-mockname";
+    public static final String TEST_PREFIX_MOCKNAME = "workspace-mockname";
```

Comment:
`TEST_PREFIX_SONDRE   != TEST_PREFIX_MOCKNAME `
Mockname needs to folllow same pattern as Sondre (with '-' )